styles.css

```diff
@@ -230,13 +230,16 @@ h2 {
   outline: none;
 }
 
-.abount-me-btn {
+.about-me-btn {
   margin-top: 24px;
+  margin-bottom: 60px;
 }
 
 .btn-type-a:hover,
```

Comment:
Great job by adding hover state to submit button. But as the previous reviewer requested, we need to consider all the states of button. Here you have missing "pressed state". Please stick to the color on Figma template. You can implement with :active selector. These states are useful to increase the user experience of your web page.